app/src/main/java/com/bogdan/codeforceswatcher/RatingUpdateReceiver.kt

```diff
@@ -0,0 +1,74 @@
+package com.bogdan.codeforceswatcher
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.graphics.Color
+import android.support.v4.app.NotificationCompat
+import android.util.Log
+import com.bogdan.codeforceswatcher.activity.MainActivity
+
+class RatingUpdateReceiver : BroadcastReceiver() {
+
+    var notificationText = ""
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        onRefresh()
+        Log.d("MeTag", "onReceive")
+        if (notificationText != "") {
+            val notificationManager = context?.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
+            val intentReceiver = Intent(context, MainActivity::class.java)
+            intentReceiver.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP
+
+            val pendingIntent = PendingIntent.getActivity(context, 0, intentReceiver, PendingIntent.FLAG_UPDATE_CURRENT)
+
+            val channelId = "1234"
+
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+                val channel = NotificationChannel(channelId, "My channel",
```

Comment:
User will see this channel name in Settings. So we can use smth more clear, like `Codeforces Wathcher`.